### PR TITLE
Change github to gitlab in section 4

### PR DESCRIPTION
**Pull request by svenvanderburg**

### CODE REVIEW

File: _episodes/41-code-review.md [hunk 1]
```diff
@@ -207,26 +207,24 @@ a fellow learner's repository.
...
 
-1. Head over to your software repository in GitHub.
-2. Navigate to the pull requests tab.
-3. Create a new pull request by clicking the green `New pull request` button.
-   ![GitHub pull requests tab](../fig/github-pull-request-tab.png){: .image-with-shadow width="900px"}
-4. Select the base and the compare branch - `main` and `feature-std-dev`, respectively.
+1. Head over to your software repository in GitLab.
+2. Navigate to the merge requests tab.
+3. Create a new merge request by clicking the blue `New merge request` button.
+   ![GitLab pull requests tab](../fig/gitlab-merge-request-tab.png){: .image-with-shadow width="900px"}
```
Comment (carschno) [inline _episodes/41-code-review.md]:
```suggestion
   ![GitLab merge requests tab](../fig/gitlab-merge-request-tab.png){: .image-with-shadow width="900px"}
```

**Review by carschno — APPROVED** [review]
Very nice! Added one suggestion.

- [ ] Update reference to a _pull_ request.

File: _episodes/41-code-review.md [hunk 2]
```diff
@@ -492,29 +487,28 @@ discussing what the best solution is in person can help defuse this.
...
 {: .challenge}
 
-### Approving a Pull Request
+### Approving a Merge Request
 
 1. Once the reviewer approves the changes, the person whose repository it is can
    merge the changes onto the base branch.
    Typically, it is the code author's responsibility to merge
    but this may differ from team to team.
    In our case, you will merge the changes on the PR on your repository.
```
Comment (sjvrijn) [inline _episodes/41-code-review.md]:
```suggestion
   In our case, you will merge the changes on the MR on your repository.
```

**Review by sjvrijn — APPROVED** [review]
One more suggestion by me, looks good after that!

**Comment by svenvanderburg** [comment]
Thanks @sjvrijn and @carschno for spotting the leftovers, this will save some confusion during the workshop!

**Pull request merged**